### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by radualex**

### CODE REVIEW

**Review by radityasurya — COMMENTED** [review]
Do we still need this htaccess?

File: app/Http/Controllers/Auth/EditController.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+use Illuminate\Foundation\Auth\User;
+
+class EditController extends Controller
+{
+    protected function validator(array $data)
+    {
+        return Validator::make($data, [
+            'name' => 'max:255',
+            'email' => 'email|max:255|unique:users',
+            'password' => 'min:6|confirmed',
```
Comment (radityasurya) [inline app/Http/Controllers/Auth/EditController.php]: I think we should add gender here

File: app/Http/Controllers/MessageController.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Validator;
+
+class MessageController extends Controller
+{
+    protected function validator(array $data)
+    {
+        return Validator::make($data, [
+            'description' => 'required|max:255',
```
Comment (radityasurya) [inline app/Http/Controllers/MessageController.php]: is description the correct variable name? i think it is better with message or content right?

File: config/constants.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+| that are better placed here rather than in .env file.
+| Use config('your_key') to get the values.
+|
+*/
+
+    'events_table' => env('EVENTS_TABLE', 'events'),
+    'eventuser_table' => env('EVENTUSER_TABLE', 'eventuser'),
+    'interests_table' => env('INTERESTS_TABLE', 'interests'),
+    'interestuser_table' => env('INTERESTUSER_TABLE', 'interestuser'),
+    'messages_table' => env('MESSAGES_TABLE','messages'),
```
Comment (radityasurya) [inline config/constants.php:26]: whats the difference between message and chat?

File: app/Http/Controllers/Auth/EditController.php (hunk 1 above)
Comment (radualex) [reply app/Http/Controllers/Auth/EditController.php]: we have to add it to migrations

File: app/Http/Controllers/MessageController.php (hunk 2 above)
Comment (radualex) [reply app/Http/Controllers/MessageController.php]: yeah..content I think it's better

File: config/constants.php (hunk 3 above)
Comment (radualex) [reply config/constants.php:26]: message is like comment inside an event. Chat is between users

**Pull request merged**